### PR TITLE
gdkx11: fix crate name

### PR DESCRIPTION
**Pull request by bilelmoussaoui**
noticed that in https://gtk-rs.org/gtk4-rs/gdk_x11_sys/index.html

### CODE REVIEW

**Review by sdroege — APPROVED** [review]

**Comment by sdroege** [comment]
Also has to be fixed for gtk(3)-x11 or is that one ok?

**Comment by bilelmoussaoui** [comment]
> Also has to be fixed for gdk(3)-x11 or is that one ok?

That one is okay, this one was mostly an error on my part when I generated gdk4-x11 bindings

**Comment by sdroege** [comment]
https://github.com/gtk-rs/gtk4-rs/pull/53/checks?check_run_id=1403353641 fails

File: gdk4-x11/Cargo.toml [hunk 1]
```diff
@@ -25,6 +25,6 @@ package = "gobject-sys"
 path = "../gdk4"
 package = "gdk4"
 
-[dependencies.gdk_x11_sys]
+[dependencies.gdk4_x11_sys]
 path = "./sys"
-package = "gdk4x11-sys"
\ No newline at end of file
+package = "gdk4-x11-sys"
```
Comment (GuillaumeGomez) [inline gdk4-x11/Cargo.toml:30]: There is a missing backline here. :)

**Review by GuillaumeGomez — APPROVED** [review]

File: gdk4-x11/Cargo.toml (hunk 1 above)
Comment (bilelmoussaoui) [reply gdk4-x11/Cargo.toml]: Fixed

**Pull request merged**